client/app/app.js

```diff
@@ -70,6 +70,10 @@ angular.module("app", [
 .run(AppLocalizeRun)
 .config(AppMaterial)
 .config(AppHTTPErrorHandler)
+.config(( $mdGestureProvider ) => {
```

Comment:
Well, I found https://github.com/angular/material/issues/1300#issuecomment-98498251  - so apperently this destorys certain swipe features of angular material, but I think having a working map is more important. Will not only be interesting for this issue, but also for clickable popups etc.